packages/server/src/hono/routes/a2a-tasks.ts

```
@@ -0,0 +1,365 @@
+        .string()
+        .optional()
+        .transform((v) => (v ? parseInt(v) : undefined)),
+    lastUpdatedAfter: z
+        .string()
+        .optional()
+        .transform((v) => (v ? parseInt(v) : undefined)),
+    includeArtifacts: z
+        .string()
+        .optional()
+        .transform((v) => v === 'true'),
+});
```

ESLint

[error] Zod method chain is missing .describe() call. Add .describe("description here") for OpenAPI documentation.

(dexto-custom/require-zod-describe)